pkg/env/env.go

```diff
@@ -4,39 +4,26 @@
 package env
 
 import (
-	"context"
-	"log"
 	"os"
 	"path"
+	"runtime"
 	"strings"
-
-	execute "github.com/alexellis/go-execute/v2"
 )
 
 // GetClientArch returns a pair of arch and os
 func GetClientArch() (arch string, os string) {
-	task := execute.ExecTask{
-		Command:     "uname",
-		Args:        []string{"-m"},
-		StreamStdio: false}
-	res, err := task.Execute(context.Background())
-	if err != nil {
-		log.Println(err)
-	}
+	arch = runtime.GOARCH
+	os = strings.ToLower(runtime.GOOS)
 
-	archResult := strings.TrimSpace(res.Stdout)
-
-	taskOS := execute.ExecTask{Command: "uname",
-		Args:        []string{"-s"},
-		StreamStdio: false}
-	resOS, errOS := taskOS.Execute(context.Background())
-	if errOS != nil {
-		log.Println(errOS)
+	if arch == "arm" {
```

Comment:
I'm not super happy with the `if`'s here. I kinda wanted to do some type of `map` to map it better and make it easier to extend in the future, but decided to start as simple as possible and improve upon request. So let me know if it is too ugly!

Reply:
Does is cover all `arm` variants?

Reply:
This code is not part of the latest commit :)